book/src/dev/integrations/fluentd.md

```diff
@@ -80,12 +80,6 @@ database [sort key](../../command/db.md#sort-key)
 to the `ts` field and sort the stored data in descending order by that key.
 This configuration is ideal for Zeek log data.
 
-> The [Zui](https://zui.brimdata.io/) desktop application automatically starts a
-> SuperDB database service when it launches. Therefore if you are using Zui you can
-> skip the first set of commands shown above. The pool can be created from Zui
-> by clicking **+**, selecting **New Pool**, then entering `ts` for the
-> [pool key](../../command/db.md#sort-key).
-
```

Comment:
We reached consensus that we'd drop mention of the desktop app in the docs for now, so I removed this text rather than convert to an admonition.